.changeset/angry-suns-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react": patch
+---
+
+ForwardRef support - CountryCode and SliderField
```

Comment:
Update the changeset maybe since forward ref for`SelectField` is included

Reply:
Done